draft-ietf-quic-http.md

```diff
@@ -476,14 +536,34 @@ how each HTTP/2 SETTINGS parameter is mapped:
   : This setting has no equivalent in QUIC.  Specifying it in the SETTINGS
     frame is an error.
 
-  SETTINGS_MAX_HEADER_LIST_SIZE
-  : Sent in SETTINGS frame.
+  SETTINGS_MAX_HEADER_LIST_SIZE:
+  : An integer with a maximium value of 2^32 - 1.
 
-As with HTTP/2, unknown SETTINGS parameters are tolerated but ignored. SETTINGS 
-parameters are acknowledged by the receiving peer, by sending an empty SETTINGS 
-frame in response with the ACK bit set.
+#### Settings Synchronization {#settings-synchronization}
 
+Some values in SETTINGS benefit from or require an understanding of when the 
+peer has received and applied the changed parameter values. In order to provide 
+such synchronization timepoints, the recipient of a SETTINGS frame MUST apply 
+the updated parameters as soon as possible upon receipt. The values in the 
+SETTINGS frame MUST be processed in the order they appear, with no other frame 
+processing between values. Unsupported parameters MUST be ignored. 
 
+Once all values have been processed, if the REQUEST_ACK flag was set, the 
+recipient MUST immediately emit a SETTINGS_ACK frame listing the identifiers 
```

Comment:
"immediately" has such a real time feel to it - while the requirement is really about ordering of when the new settings can have an impact on the semantics of future I/O.. it should probably be written like that